### DISCUSSION TITLE
Prisma not working in the browser (SvelteKit project, MongoDB)

### DISCUSSION THREAD

**Discussion by TheOnlyTails** [prisma/prisma#14162] [Q&A]
This is my first time using Prisma; I've connected it to a mongodb database, and set up the schema.
However, the moment I try to run the project, I get this error in the browser console:
```
Uncaught (in promise) Error: PrismaClient is unable to be run in the browser.
```
I'm defining the client like so in an external js file:
```ts
// src/lib/data.ts
import { PrismaClient } from "@prisma/client"

export const client = new PrismaClient({
	errorFormat: "pretty",
})
```

**Comment by ruheni** [comment]
Hi 👋🏽

Prisma runs on the server-side, and it seems you're running it on the browser/ client-side. We're working on a Prisma Example with SvelteKit (WIP) with the help of @joshnuss. Before we publish the example, feel free to take a look at [this PR](https://github.com/prisma/prisma-examples/pull/3907) to learn how you can integrate Prisma and SvelteKit. The default example uses SQLite, but you can swap it out with your preferred provider, i.e., MongoDB. 

Let me know if you run into any issues. 🙂

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
